components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksUseCases.kt

```diff
@@ -83,6 +77,24 @@ class AppLinksUseCases(
             .toHashSet()
     }
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun getBrowserPackageNames(): Set<String> {
+        if (browserPackageNames != null) {
+            return browserPackageNames
+        }
```

Comment:
Unrelated to the change here, but I wonder what the use case for passing those in is? 🤔

Reply:
This was to allow caller to provide their own list of browser package names so we don't need to detect it.  I think this was introduced mainly for testing.

Reply:
For testing only that is a bit weird though since that would shortcut the code here and we would test something else. :)

Reply:
I agree.  I will create an issue to remove this from the API.

Reply:
https://github.com/mozilla-mobile/android-components/issues/7229